### PR TITLE
[Playground] Feature: Headless components

### PR DESCRIPTION
**Pull request by kien-ngo**
CNCT-2588

---

## PR-Codex overview
This PR focuses on enhancing the `playground-web` application by adding new components for NFT and account functionalities, improving user interface elements, and integrating headless UI components for seamless wallet interactions.

### Detailed summary
- Added `ArticleCardIndex` for displaying information about `Headless UI`.
- Introduced new navigation links for `Headless UI`.
- Expanded `Account` and `NFT` components with additional features and examples.
- Created new pages for `NFT Components` and `Account Components`.
- Enhanced query parameters for `useQuery` in account-related components.
- Developed multiple examples for `NFT` and `Account` components, showcasing usage and customization options.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5535?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5535** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5535?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5535?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#5533** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5533?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#5457** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5457?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Comment by kien-ngo** [comment]
DEVX-1831

**Comment by kien-ngo** [comment]
CNCT-2588

File: apps/playground-web/src/app/connect/ui/nft/page.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  NftMediaBasic,
+  NftNameBasic,
+} from "@/components/headless-ui/nft-examples";
+import ThirdwebProvider from "@/components/thirdweb-provider";
+import { metadataBase } from "@/lib/constants";
+import type { Metadata } from "next";
+
+export const metadata: Metadata = {
+  metadataBase,
+  title: "lorem ipsum",
```
Comment (joaquim-verges) [inline apps/playground-web/src/app/connect/ui/nft/page.tsx:14]: can you fix these pls?

File: apps/playground-web/src/components/headless-ui/account-examples.tsx [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+          <AccountName
+            className="text-left"
+            loadingComponent={
+              <AccountAddress
+                formatFn={shortenAddress}
+                className="text-left"
+              />
+            }
+            fallbackComponent={
+              <AccountAddress
```
Comment (joaquim-verges) [inline apps/playground-web/src/components/headless-ui/account-examples.tsx:477]: wonder if this should be the fallback by default. feels like most ppl would want that

File: apps/playground-web/src/app/connect/ui/nft/page.tsx (hunk 1 above)
Comment (kien-ngo) [reply apps/playground-web/src/app/connect/ui/nft/page.tsx]: updated

File: apps/playground-web/src/components/headless-ui/nft-examples.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  chain: ethereum,
+  client: THIRDWEB_CLIENT,
+});
+
+export function NftMediaBasic() {
+  return (
+    <>
+      <div className="space-y-2">
+        <h2 className="font-semibold text-2xl tracking-tight sm:text-3xl">
+          NFTImage
```
[the remark this answers is not in the document]
Comment (kien-ngo) [reply apps/playground-web/src/components/headless-ui/nft-examples.tsx:20]: fixed

File: apps/playground-web/src/components/headless-ui/account-examples.tsx (hunk 2 above)
Comment (kien-ngo) [reply apps/playground-web/src/components/headless-ui/account-examples.tsx:477]: Does making it return a different component, makes it impure? Personally I'm happy with making things a bit redundant (or verbose, if you will) in return for what we call a true headless component

**Review by joaquim-verges — APPROVED** [review]
looking good ser!

**Pull request merged**